src/main/java/seedu/address/MainApp.java

```diff
@@ -47,9 +47,9 @@
     protected Ui ui;
     protected Config config;
     protected UserPrefs userPrefs;
-    protected AnakinLogic logic;
-    protected AnakinModel model;
-    protected Storage storage;
+    protected Logic logic;
+    protected Model model;
+    protected AnakinStorage anakinStorage;
```

Comment:
Kind of weird that the main app components are `Logic`, `Model`, `UI`, but for storage it's `AnakinStorage`.. i don't think you should've swapped the `AnakinStorage` and `Storage` interfaces